Codes/Clean Data/getAggressiveness.R

```diff
@@ -0,0 +1,22 @@
+function(direction,

```

Comment:
Interesting to know you can indeed define R function in this way. I always think the R function must be defined at this way
```R
functionName <- function(arg1, arg2) {
...
}
```
A benefit of this is that we can put multiple function definitions in one file. I wonder what is the benefit of defining function names by the file name. -- Indeed, many people do not like Matlab (and Java) which force programmer to put one function (Class/Interface in Java) in one file

Reply:
As we discussed today on Skype, I keep the functions in separate files to more easily use them across multiple code. However, I can see the benefit of keeping the number of subfiles to a minimum!